src/main/java/org/spongepowered/api/effect/particle/ParticleTypes.java

```diff
@@ -129,15 +126,15 @@
      */
     public static final Supplier<ParticleType> FERTILIZER = Sponge.getRegistry().getCatalogRegistry().provideSupplier(ParticleType.class, "fertilizer");
 
-    /**
-     * This particle type will play a fireworks effect.
-     *
-     * <p>This type has limited {@link ParticleOption}s, only
-     * {@link ParticleOptions#FIREWORK_EFFECTS} is supported.</p>
-     */
-    public static final Supplier<ParticleType> FIREWORKS = Sponge.getRegistry().getCatalogRegistry().provideSupplier(ParticleType.class, "fireworks");
+//    /** TODO
+//     * This particle type will play a fireworks effect.
+//     *
+//     * <p>This type has limited {@link ParticleOption}s, only
+//     * {@link ParticleOptions#FIREWORK_EFFECTS} is supported.</p>
+//     */
+//    public static final Supplier<ParticleType> FIREWORKS = Sponge.getRegistry().getCatalogRegistry().provideSupplier(ParticleType.class, "fireworks");
 
-    public static final Supplier<ParticleType> FIREWORKS_SPARK = Sponge.getRegistry().getCatalogRegistry().provideSupplier(ParticleType.class, "fireworks_spark");
+    public static final Supplier<ParticleType> FIREWORK = Sponge.getRegistry().getCatalogRegistry().provideSupplier(ParticleType.class, "firework");
```

Comment:
How are you going to name the original `FIREWORKS` if the spark becomes `FIREWORK`?

Reply:
Another option is to make the type work as a firework effect, when the `FIREWORK_EFFECTS` option is present, and otherwise show the spark.